lib/portage/dbapi/bintree.py

```diff
@@ -420,9 +426,16 @@ async def unpack_contents(self, pkg, dest_dir, loop=None):
                 if extractor.returncode != os.EX_OK:
                     raise PortageException(f"Error Extracting '{pkg_path}'")
             elif binpkg_format == "gpkg":
+                repoconfig = self.bintree.get_local_repo(cpv)
+                if not repoconfig:
+                    # XXX: ???
```

Comment:
I only saw this case when I wasn't using a separate `location` so it may not matter, need to check.